CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 Release notes for changes to the trainer
+---
+**4.8.1**
```

Comment:
Here is a wrong version of the game. You put 4.8.2 in Main.cs file but here remained an old one.

Reply:
Corrected the changelog. Made the UltraEfficiencyModifier adjustable.